feat(ecs): `interactive` option in `ContainerDefinitionOptions` (#28536)
aws/aws-cdk: packages/aws-cdk-lib/aws-ecs/README.md

taskDef.grantRun(role);
```

> To deploy containerized applications that require the allocation of standard input (stdin) or a terminal (tty), you can set the `interactive` propertiy to `true` in `ContainerDefinitionOptions`.

Reviewer: ```suggestion
To deploy containerized applications that require the allocation of standard input (stdin) or a terminal (tty), use the `interactive` property.
```

> This parameter corresponds to OpenStdin in the [Create a container](https://docs.docker.com/engine/api/v1.35/#tag/Container/operation/ContainerCreate) section of the [Docker Remote API](https://docs.docker.com/engine/api/v1.35/) and the `--interactive` option to `docker run`.

Reviewer: ```suggestion
This parameter corresponds to `OpenStdin` in the [Create a container](https://docs.docker.com/engine/api/v1.35/#tag/Container/operation/ContainerCreate) section of the [Docker Remote API](https://docs.docker.com/engine/api/v1.35/)
and the `--interactive` option to [docker run](https://docs.docker.com/engine/reference/run/#security-configuration).
```
Let's add also the link to `docker run`.
